lib/src/multipart_body.dart

```diff
@@ -8,19 +8,17 @@ final _random = Random();
 
 class MultipartBody implements RequestBody {
   final List<Part> parts;
-  MediaType _contentType;
+  MediaType type;
```

Comment:
should use "_contentType" as it was before

```suggestion
  MediaType _contentType;
```